src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,13 @@
 const accessibilityQuiz = [
+  {
+    Question: "What legal repercussions can businesses face for failing to meet website accessibility requirements?",
+    Answer: "Businesses that fail to meet website accessibility requirements may face lawsuits, financial penalties, and reputational damage, in addition to the potential loss of customers and legal compliance obligations.",
```

Comment:
the answer is to long.
remember that is needs to fit in the answer container on the site
https://developerquiz.org/

```suggestion
    Answer: "Businesses can face lawsuits or financial penalties",
```